adapters/adform/adform.go

```diff
@@ -57,6 +58,8 @@ type adformDigitrustPrivacy struct {
 type adformAdUnit struct {
 	MasterTagId json.Number `json:"mid"`
 	PriceType   string      `json:"priceType,omitempty"`
+	KeyValues   string      `json:"mkv,omitempty"`
+	Keywords    string      `json:"mkw,omitempty"`
```

Comment:
Can you please use the mixed caps naming convention here, i.e., KeyWords instead of Keywords?  This change will affect some of the code below, too,